.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: java
+
+# This project is aimed to be JDK7-compatible; however, there are
+# issues with testing against it.
+# openjdk7: https://github.com/gradle/gradle/issues/2421
+# oraclejdk: https://github.com/travis-ci/travis-ci/issues/7884
+jdk:
+  - oraclejdk8
+
+# As recommended in:
+# https://docs.travis-ci.com/user/languages/java/#Caching
+env:
+  global:
+  - secure: "pDsnrAyqSszuOPdkGKSxGrypbPv6obtQ3BjJsLcWc09YwuSLK8VKfvhbL5KdqvtJB43j+5w3xnpjCAZKLLECLym96n6UUCDjq5wr+RC5LBTBzTdMuKBUmXLWbQLRzC/+i9Z25k5UeIhJTAE/dDnZPQnpNv0sxviTf/p1K7BKfEBqhZsVtAPTxkrdghhE5muOO6lH9xMazKxawVAaNVqzaKnbzVpntXBFtLlr5E5W97fqb9CCFf1fn33e5KZAclNPtKOH7R7NMStYZ+hKp9N93dEkUYnyK3f9/arwbrW/F66cDAqYmvxkIQ8ghAf0/0TU9gZF3Y/PlOWqOFa9Bg87yMOeelGwCPFgw0yJTtIFbnMD26oP3ben+/XZoOCBPz9cO1Hy0P9kADaYYPw64ahETKYrUhizDoy/8x2O1WSsn/mCLWtdaWXyQZ4AboNbxb7bAs1JJxm/cCqUa/CheOqxUUAWD4J4KSiTAdn2cW/84aowYhsbWIkN/JyNFCzGZ0apS5DVAXOuRyZ31UdjXrfK2UP2ZIjrfvYfzsjepc9uTkqHpjMX5U1xRdPlpiZSecToP+FDUq6t8DudQ0SrIc7fNg7H6fXSvtlpMjCIOaUfkt7BReaNXn8bWJVXU1Nd8VEwbYptoUwQxK/Pu9sEkVsAJijbmaIaCmaJnLyClrgxJmg="
```

Comment:
could you put a note about how devs should use/update/etc these in `Developers.md`